app/config/config.yml

```diff
@@ -295,6 +295,13 @@ old_sound_rabbit_mq:
             callback:          AppBundle\Consumer\ProjectCitizenCreationNotificationConsumer
             qos_options:       { prefetch_size: 0, prefetch_count: 1, global: false }
 
+        mailer:
+            connection:        default
+            exchange_options:  { name: 'mails', type: direct }
+            queue_options:     { name: 'mails', durable: false, routing_keys: [mailer.scheduled_mail]}
```

Comment:
Why do we need to specify a routing key ? it could be anything no ?

Reply:
removed